apps/charterafrica/src/payload/collections/Explainers.js

```diff
@@ -0,0 +1,53 @@
+import richText from "../fields/richText";
+
+const Explainer = {
+  slug: "explainers",
+  admin: {
+    useAsTitle: "title",
+  },
+  access: {
+    read: () => true, // Everyone can read Media
+  },
+  fields: [
+    {
+      name: "title",
+      label: {
+        en: "Title",
+        fr: "Titre",
+        pt: "Título",
+      },
+      type: "text",
+      localized: true,
+      required: true,
+    },
+    richText({
+      name: "description",
+      label: {
+        en: "Description",
+        fr: "La description",
+        pt: "Descrição",
+      },
+      admin: {
+        elements: [],
+      },
+      localized: true,
+      required: true,
+    }),
+    {
+      name: "image",
+      type: "upload",
+      relationTo: "media",
+      required: true,
```

Comment:
Thinking out loud, since these are images, aren't we going to have different image per language?